docs/source/api.rst

```diff
@@ -25,9 +25,36 @@ Ensemble Operator Constructors
 
    workflow.TransformWorkflow
    tensorflow.PredictTensorflow
+   fil.PredictForest
+   implicit.PredictImplicit
    softmax_sampling.SoftmaxSampling
    session_filter.FilterCandidates
    unroll_features.UnrollFeatures
 
 .. faiss.QueryFaiss
 .. feast.QueryFeast
+
+
+Persist Workflows and Ensembles
+-------------------------------
+
+.. currentmodule:: merlin.systems.triton
+
+.. autosummary::
+   :toctree: generated
+
+   export_tensorflow_ensemble
+   export_pytorch_ensemble
+   export_hugectr_ensemble
```

Comment:
I don't think we actually want to include these three functions in the docs, since they're legacy code inherited from NVT that we'd like to get rid of 😅

Reply:
NP, I'll ditch them.